testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -137,3 +138,16 @@ Feature: Smoke tests for <client>
     When I force picking pending events on "<client>" if necessary
     And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
     Then I should see a "This action's status is: Completed" text
+
+  Scenario: Install spacecmd from the client tools on the <client>
+    Given I am on the Systems overview page of this "<client>"
+    When I follow "Software" in the content area
+    And I follow "Install"
+    And I enter "spacecmd" as the filtered package name
```

Comment:
I wonder if we can replace our package install scenario, by this one :thinking: 
This is installing something from client tools, so better keep both?

Reply:
no we can't i already had same idea and discussed it with julio

yes because of the client tools thingie, and also because it tests installation still works after reboot